### DISCUSSION TITLE
How to blend the dataset in function and generative data to on dataset?

### DISCUSSION THREAD

**Discussion by jiaruHithub** [pyg-team/pytorch_geometric#6711] [Q&A] [answered]
I am working on a graph classification problem. Based on IDMB dataset, I have generated some graphs with labels. How to mix those graphs with graphs from IDMB in one dataset ? Or how to use ```dataloader``` func to load them together ? 

Expect result: Every batch has some graphs from IDBM and generative graphs.

Thanks!

**Comment by rusty1s** [comment, marked as answer]
You can just put them into a list of `Data` objects and input them into your loader:
```python
data_list = [data for data in dataset]
data_list.extend(generated_data_objects)

loader = DataLoader(data_list, batch_size=128, shuffle=True)
